zppy/__main__.py

```diff
@@ -29,15 +29,23 @@ def main():
     templateDir = os.path.join(os.path.dirname(__file__), "templates")
 
     # Read configuration file and validate it
-    config = ConfigObj(args.config, configspec=os.path.join(templateDir, "default.ini"))
-    validator = Validator()
-
-    result = config.validate(validator)
-    if result is not True:
-        print("Validation results={}".format(result))
-        raise Exception("Configuration file validation failed")
+    default_config = os.path.join(templateDir, "default.ini")
+    user_config = ConfigObj(args.config, configspec=default_config)
+    _validate_config(user_config)
+    campaign = user_config["default"]["campaign"]
+    if campaign != "custom":
```

Comment:
Perhaps a better default `campaign` would be `"default_settings"`, since this setting simply tells `zppy` to use the `default.ini` values for `sets` and `generate`.

Reply:
I think I would find `campaign = "none"` to be a more intuitive default.  Since the defaults are now (and probably will continue to be) the same as the `water_cycle`, we could just make `water_cycle` the default.